hw/ip/entropy_src/data/entropy_src.hjson

```diff
@@ -137,122 +153,459 @@
+        { bits: "31:0",
+          name: "REPCNT_TOTAL_FAILS",
+          desc: '''This register will hold a running count of test failures observed
+                   during normal operation. It will persist until cleared.
                 '''
-          resval: "0x00000004"
-      }
+        }
       ]
-    }
-    { name: "ES_SEED",
+    },
+    { name:     "ADAPTP_HI_TOTAL_FAILS",
+      desc:     "Adaptp high test failure counter register",
+      swaccess: "ro",
+      hwaccess: "hwo",
+      hwext: "true",
+      fields: [
+        { bits: "31:0",
+          name: "ADAPTP_HI_TOTAL_FAILS",
+          desc: '''This register will hold a running count of test failures observed
+                   during normal operation. It will persist until cleared.
+                '''
+        }
+      ]
+    },
+    { name:     "ADAPTP_LO_TOTAL_FAILS",
+      desc:     "Adaptp low test failure counter register",
+      swaccess: "ro",
+      hwaccess: "hwo",
+      hwext: "true",
+      fields: [
+        { bits: "31:0",
+          name: "ADAPTP_LO_TOTAL_FAILS",
+          desc: '''Tlos register will hold a running count of test failures observed
+                   during normal operation. It will persist until cleared.
+                '''
+        }
+      ]
+    },
+    { name:     "BUCKET_TOTAL_FAILS",
+      desc:     "Bucket test failure counter register",
+      swaccess: "ro",
+      hwaccess: "hwo",
+      hwext: "true",
+      fields: [
+        { bits: "31:0",
+          name: "BUCKET_TOTAL_FAILS",
+          desc: '''This register will hold a running count of test failures observed
+                   during normal operation. It will persist until cleared.
+                '''
+        }
+      ]
+    },
+    { name:     "MARKOV_TOTAL_FAILS",
+      desc:     "Markov test failure counter register",
+      swaccess: "ro",
+      hwaccess: "hwo",
+      hwext: "true",
+      fields: [
+        { bits: "31:0",
+          name: "MARKOV_TOTAL_FAILS",
+          desc: '''This register will hold a running count of test failures observed
+                   during normal operation. It will persist until cleared.
+                '''
+        }
+      ]
+    },
+    { name: "ALERT_THRESHOLD",
+      desc: "Alert threshold register",
+      swaccess: "rw",
+      hwaccess: "hro",
+      regwen:   "REGEN",
+      fields: [
+        { bits: "3:0",
+          name: "ALERT_THRESHOLD",
+          desc: '''This is the threshold size that will signal an alert when
+                   value is reached. A value of zero will disable alerts.
+                   The default value is 2.
+                '''
+          resval: "0x0002"
+        }
+      ]
+    },
+    { name:     "ALERT_FAIL_COUNTS",
+      desc:     "Alert failure counts register",
+      swaccess: "ro",
+      hwaccess: "hwo",
+      hwext: "true",
+      fields: [
+        { bits: "3:0",
+          name: "ANY_FAIL_COUNT",
+          desc: '''This register will hold a running count of
```

Comment:
Is this intended to be the sum of all the other count bitfields?  I think so but maybe put explicit language here if that's true.  (And if I'm correct, is this wide enough?  Might need to be 7:0 instead?)